tests/foreman/api/test_template.py

```diff
@@ -110,7 +110,7 @@ def tftpboot(module_org, target_sat):
         },
     }
     # we keep the value of these for the teardown
-    default_settings = entities.Setting().search(query={"search": "name ~ Global default"})
+    default_settings = entities.Setting().search(query={"search": "name ~ global_"})
```

Comment:
It would be a good idea to write a more exact search to filter out other settings that begin with `global`.
```suggestion
    default_settings = [s for s in entities.Setting().search(query={"search": "name ~ global_"}) if 'PXE' in s.name]
```

We are unable to do so using the search query now. I filed a BZ for that a few minutes back: https://bugzilla.redhat.com/show_bug.cgi?id=2090799